jgiven-android/build.gradle

```diff
@@ -57,18 +59,12 @@ dependencies {
     api "com.google.guava:guava:$guavaAndroidVersion"
     api("net.bytebuddy:byte-buddy-android:$byteBuddyVersion")
     api "junit:junit:$junitVersion"
-    implementation "com.android.support:appcompat-v7:$androidBuildToolsVersion"
-    implementation('com.android.support.test:rules:0.5') {
+    implementation "androidx.appcompat:appcompat:1.3.1"
+    implementation('androidx.test:core:1.4.0')
+    implementation('androidx.test:rules:1.4.0') {
         exclude group: "junit"
     }
-    //implementation files('libs/dx-1.7.jar')
-    implementation(project(path: ':jgiven-junit')) {
-        exclude group: "com.google.guava"
-        exclude group: "net.bytebuddy"
-    }
-    implementation(project(path: ':jgiven-core')) {
-        exclude group: "com.google.guava"
-        exclude group: "net.bytebuddy"
-    }
+    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
+    api project(":jgiven-core")
```

Comment:
please go check the build.gradle of jgiven-junit. It should actually suffice to have it as a dependency, because jgiven-junit should already disclose jgiven-core as its dependency